content.js

```
@@ -41,39 +41,111 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+              });
+              
+              if (response && !response.startsWith("Translation error:")) {
+                element.textContent = response;
+                element.classList.add('hinglish-translated');
+                translatedCount++;
+              }
+            } catch (error) {
+              console.error('Translation error:', error);
+            }
+            resolve();
+          });
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)